web/s_keyman_com.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
```

Comment:
Could we put this script within a subfolder?

For reference, the iOS platform now has a CI script at `ios/tools`.

Reply:
Possibly. Though `web/tools/` currently has recorder test pages